packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -108,6 +108,13 @@ import {
   S3BucketCard,
   LambdaFunctionCard,
   IAMUserCard,
+  IAMRoleCard,
```

Comment:
@sblausten this adds a new card to the entity page with the details of an iam role if there is an annotation on the entity "amazon.com/iam-role-arn"